### PR TITLE
 Added Board

### PR DESCRIPTION
**Pull request by IStallcup**
Added a beginning board class file and some tests.

### CODE REVIEW

File: Assignment1/src/main/java/controllers/board.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        //initializes columns
+        addFourColumns();
+    }
+
+    //add the four columns to the linked list
+    public void addFourColumns()
+    {
+        LinkedList<card> firstCol = new LinkedList<>();
+        boardList.add(firstCol);
+        boardList.add(firstCol);
```
Comment (athai) [inline Assignment1/src/main/java/controllers/board.java]: There are four columns but they're all firstCol! Rename some.

File: Assignment1/src/test/java/controllers/boardTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        playBoard.addCardToColumn(1,playBoard.getCardFromDeck());
+        playBoard.addCardToColumn(2,playBoard.getCardFromDeck());
+        playBoard.addCardToColumn(3,playBoard.getCardFromDeck());
+        assert playBoard.boardList.get(0).peekLast().num == 0;
+        assert playBoard.boardList.get(1).peekLast().num == 0;
+        assert playBoard.boardList.get(2).peekLast().num == 0;
+        assert playBoard.boardList.get(3).peekLast().num == 0;
+    }
+
+    @Testt
```
Comment (athai) [inline Assignment1/src/test/java/controllers/boardTest.java]: Test is spelled wrong.

File: Assignment1/src/main/java/controllers/board.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    //adds a custom card to the column colToAddTo
+    public void addCardToColumn(int colToAddTo, card cardToAdd) {
+        boardList.get(colToAddTo).add(cardToAdd);
+    }
+
+    //checks the first card in inputted columns for number removability (will be expanded to check top card in column)
+    public boolean checkNums(int checkCol, int removeCol)
+    {
+        if (boardList.get(0).get(0).num > boardList.get(0).get(1).num)
+            return true;
```
Comment (athai) [inline Assignment1/src/main/java/controllers/board.java]: Syntax!

**Comment by leel8on** [comment]
Why are these in controllers rather than models?

**Comment by IStallcup** [comment]
Fixed Aileen's issues and Lee's issues as far as I can tell!

**Comment by leel8on** [comment]
+1

**Comment by kannabanana** [comment]
I think you're missing a deconstructor and a copy constructor for your linked list of linked list :+1:

**Comment by IStallcup** [comment]
Oops! Reopening this. Github (and Isaac) are silly!

**Comment by IStallcup** [comment]
Okay, removed all personal useless work on deck, card so it's not in the way of anyone. As of right now this is most current board and boardTest work!

File: Assignment1/src/test/java/models/boardTest.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    {
+        board playBoard = new board();
+        card addCard = new card(2,"hearts");
+        playBoard.addCardToColumn(0,addCard);
+        assert playBoard.boardList.get(0).peekFirst().num == 2;
+        assert playBoard.boardList.get(0).peekFirst().suit == "hearts";
+    }
+
+    @Test
+    public void testAddCarToTop()
```
Comment (athai) [inline Assignment1/src/test/java/models/boardTest.java]: Add car != add card

Is this a test for the deal function?

**Comment by leel8on** [comment]
Looks good :+1:

**Comment by athai** [comment]
lgtm. +1

**Pull request merged**